samples/copy_file.bal

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/log;
+import ballerina/os;
+import ballerinax/googleapis_drive as drive;
+
+configurable string clientId = os:getEnv("CLIENT_ID");
+configurable string clientSecret = os:getEnv("CLIENT_SECRET");
+configurable string refreshToken = os:getEnv("REFRESH_TOKEN");
+configurable string refreshUrl = os:getEnv("REFRESH_URL");
+
+string sourceFileId = "<PLACE_YOUR_FILE_ID_HERE>";
+
+###################################################################################
+# Copy file by ID
+###################################################################################
+# Creates a copy of a file and applies any requested updates with patch semantics. 
+# **Folders cannot be copied
+# ################################################################################
+# More details : https://developers.google.com/drive/api/v3/reference/files/copy
+# #################################################################################
+# 
+
+public function main() {
+    drive:Configuration config = {
+        clientConfig: {
+            clientId: clientId,
+            clientSecret: clientSecret,
+            refreshUrl: refreshUrl,
+            refreshToken: refreshToken
+        }
+    };
+    drive:Client driveClient = new (config);
+    drive:File|error res = driveClient->copyFile(sourceFileId);
+    // drive:File|error response = driveClient->copyFile(sourceFileId, destinationFolderId);
+    // drive:File|error response = driveClient->copyFile(sourceFileId, destinationFolderId, newFileName);
+    //Print file ID
+    if(res is drive:File){
+        string id = res?.id.toString();
+        log:print(id);
+    } else {
+        log:printError(res.message());
+    }
+    
```

Comment:
Remove unwanted newlines in all the places

Reply:
Thanks @LakshanSS 
removed